response_operations_ui/controllers/message_controllers.py

```diff
@@ -33,3 +33,20 @@ def get_message_list(params):
     except KeyError:
         logger.exception("Response was successful but didn't contain a 'messages' key")
         raise NoMessagesError
+
+
+def send_message(message_json):
+    logger.debug("Sending message")
+
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/send-message'
+    # This will be removed once UAA is completed.  For now we need the call to backstage to include
+    # an Authorization in its header a JWT that includes party_id and role.
+    encoded_jwt = jwt.encode({'user': 'BRES', 'party_id': 'BRES', 'role': 'internal'}, 'testsecret', algorithm='HS256')
```

Comment:
Is there a reason this is still hardcoded to BRES?

Reply:
Until a new auth server is in place this is the only way to work with secure message